### PR TITLE
docs: add sass installation information

### PR DESCRIPTION
**Pull request by ricardovanlaarhoven**
## Description
After following the installation guide sass is missing as an dependency. When installed globally this doesn't seem to be an issue. But for some coworkers of mine it was. Also in ci/cdlike githubactions this was an issue as wel.

```
vite v3.1.3 building for production...
transforming...
✓ 204 modules transformed.
[vite:css] Preprocessor dependency "sass" not found. Did you install it?
file: /home/runner/work/mobyyou-clubportal/mobyyou-clubportal/src/App.vue?vue&type=style&index=0&scoped=a9d3ab21&lang.scss
error during build:
Error: Preprocessor dependency "sass" not found. Did you install it?
```

## How Has This Been Tested?
not

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by ricardovanlaarhoven** [comment]
It's more or less also created as an discussion. Perhaps it should be in the vuetify cli plugin?

**Comment by KaelWD** [comment]
Or on https://next.vuetifyjs.com/en/features/sass-variables/, basic usage is precompiled css and doesn't need sass

**Comment by ricardovanlaarhoven** [comment]
I'm not doing anything with sass variables at the moment. It's just a simple vue3 vite project with vuetify added. And i'm getting this error.

Am I doing something wrong?

**Comment by ricardovanlaarhoven** [comment]
Ooh, vueitfy doesn't need it by default. But i am using it in my own modules.. ofcource..

File: packages/docs/src/pages/en/features/sass-variables.md [hunk 1]
```diff
@@ -15,6 +15,31 @@ Vuetify uses **SASS/SCSS** to craft the style and appearance of all aspects of t
 
 <entry />
 
+## installation
```
Comment (KaelWD) [inline packages/docs/src/pages/en/features/sass-variables.md:18]:
```suggestion
## Installation
```

File: packages/docs/src/pages/en/features/sass-variables.md [hunk 2]
```diff
@@ -15,6 +15,31 @@ Vuetify uses **SASS/SCSS** to craft the style and appearance of all aspects of t
...
+
+When you do not use SASS/SCSS or vuetify's variables yourself, you don't have to install it.
+Vuetify works out of the box. But when you do when you want to change SASS/SCSS variables or use it yourself.
```
Comment (KaelWD) [inline packages/docs/src/pages/en/features/sass-variables.md:21]:
```suggestion
Vuetify works out of the box without any additional compilers needing to be installed. Changing or using SASS variables though obviously requires the SASS compiler.
```

File: packages/docs/src/pages/en/features/sass-variables.md [hunk 3]
```diff
@@ -15,6 +15,31 @@ Vuetify uses **SASS/SCSS** to craft the style and appearance of all aspects of t
...
+
+### vite
+
+Vite does provide built-in support for .scss, .sass, .less, .styl and .stylus files. There is no need to install Vite-specific plugins for them, but the corresponding pre-processor itself must be installed. Vuetify needs sass as a preprocessor so install it:
```
Comment (KaelWD) [inline packages/docs/src/pages/en/features/sass-variables.md:25]:
```suggestion
Vite provides built-in support for sass, less and stylus files. There is no need to install Vite-specific plugins for them, but the corresponding pre-processor itself must be installed. Vuetify needs sass as a preprocessor so install it:
```

**Pull request merged**